### DISCUSSION TITLE
Discussions Topics for Nov 22, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#3144] [Show and tell]
1. @mauricereeves SOAP transport
2. @rhood23699 tables management and design

**Comment by MauriceReeves-usds** [comment]
Here is the PA for the SOAP transport and the PA Redox migration: https://github.com/CDCgov/prime-reportstream/pull/2961

**Comment by carlosfelix2** [comment]
@carlosfelix2 LIVD questions
